core/Settings.js

```diff
@@ -150,11 +150,11 @@ export default class Settings {
 
     _init() {
         this.mainBlock = new UIRoundedRectangle(5)
-            .setX((20).percent())
-            .setY((20).percent())
-            .setWidth((60).percent())
-            .setHeight((50).percent())
-            .setColor(ElementUtils.getJavaColor(this.handler.getColorScheme().Amaterasu.backgroundBox))
+            .setX((this.handler.getColorScheme().Amaterasu.backgroundBoxSize[0]).percent())
```

Comment:
we can find a better way of doing this

Reply:
For sure i was kinda just getting the idea down and working first

Reply:
it's a great idea, maybe we make it a method ?
say: `Settings(...etc).setSize(width, height)` also position